Duckling/Time/KA/Corpus.hs

```diff
@@ -77,8 +74,6 @@ allExamples = concat
              ]
   , examples (datetime (2013, 2, 18, 0, 0, 0) Day)
              [ "ორშაბათი"
-             , "ორშ."
```

Comment:
We removed some examples, which were just short forms of months and weekdays, because when adding new rules to make engine more comprehensible, based on characteristics of Georgian language, they were incorrectly fitted as prefixes and led to wrong behavior.